sasl-plain-access-token/README.md

```diff
@@ -0,0 +1,13 @@
+# SASL/PLAIN Access Token Client Examples
+
+This directory contains client libraries and examples on showing how to use SASL/Plain authentication with GMK using [Access Tokens](https://cloud.google.com/docs/authentication/token-types#access) - as an alternative to using base64 encoded Service Account Keys.
```

Comment:
pedantic comment: I'd say this directory contains client library **extensions**.

Reply:
Done.